src/uffd_handler/src/main.rs

```diff
@@ -95,6 +120,16 @@ fn build_arg_parser() -> ArgParser<'static> {
                 .takes_value(true)
                 .help("The path to the snapshot memory file to serve page faults from."),
         )
+        .arg(
+            Argument::new("before-kib")
```

Comment:
Mh, if this always needs to be a multiple of the page size, would it make sense to specify this argument in "number of pages to prefault" as opposed to "kb to prefault"?